ratis-server-api/src/main/java/org/apache/ratis/server/DivisionInfo.java

```diff
@@ -51,6 +53,12 @@ default boolean isListener() {
   /** Is this server division currently the leader and ready? */
   boolean isLeaderReady();
 
+  /**
+   * @return return the id of the current leader if the leader is known to this server division;
```

Comment:
The second "return" should be removed; see https://www.oracle.com/hk/technical-resources/articles/java/javadoc-tool.html

Reply:
Fixed